h5p-bildetema-words-topic-image/src/h5p/H5PWrapper.tsx

```diff
@@ -17,7 +17,7 @@ export type Params = {
     subTopicId: string;
   };
   l10n: Record<TranslationKey, string>;
-  activeLanguage?: LanguageCode;
+  currentLanguage?: LanguageCode;
```

Comment:
Do we need to change this in semantics as well?

Reply:
"activeLanguage" is not used in semantics(?)

Reply:
I see.